lib/propolis/src/enlightenment/hyperv/overlay.rs

```diff
@@ -1063,4 +1063,46 @@ mod test {
 
         ctx.assert_pfn_has_fill(pfn, 0x70);
     }
+
+    #[test]
+    fn export_import_round_trip() {
```

Comment:
take it or leave it: this type of "round trip" test always feels like kind of a good candidate for a property test where we generate a bunch of random things and assert they look the same on both sides. but, i'm not sure if it's really going to gain that much from it, and writing the proptest code to generate arbitrary migration payloads might be too much work...just a thought!